wsf_scraping/settings.py

```diff
@@ -23,7 +23,7 @@
 # LOG_ENABLED = False
 LOG_LEVEL = 'INFO'
 LOG_FORMATTER = 'wsf_scraping.middlewares.PoliteLogFormatter'
-LOG_FILE = 'var/log-{log_level}.txt'.format(log_level=LOG_LEVEL)
+# LOG_FILE = 'var/log-{log_level}.txt'.format(log_level=LOG_LEVEL)
```

Comment:
remove if not needed

Reply:
I though I already deleted it 🤔 
I'll delete it for good this time 👍